src/swaps/liquality/LiqualitySwapProvider.js

```diff
@@ -504,7 +505,8 @@ class LiqualitySwapProvider extends SwapProvider {
       BNB: 45000,
       MATIC: 45000,
       NEAR: 8000000000000,
-      ERC20: 100000
+      ERC20: 100000,
+      ARBETH: 2000000
```

Comment:
Is this correct? It doesn't seem right that the claiming takes the same amount as gas as for initiating